Library/Formula/sqliteodbc.rb

```diff
@@ -0,0 +1,25 @@
+class Sqliteodbc < Formula
+  desc "SQLite ODBC driver"
+  homepage "http://www.ch-werner.de/sqliteodbc/"
+  url "http://www.ch-werner.de/sqliteodbc/sqliteodbc-0.9992.tar.gz"
+  sha256 "f5c6119cfb688fc3d2fbc500c6cfa796538b767c511be5ab90fb6805fcd289c3"
+
+  depends_on "sqlite"
+  depends_on "unixodbc"
+
+  def install
+    lib.mkdir
+    system "./configure", "--prefix=#{prefix}"
+    system "make"
+    system "make", "install"
+    File.rename "#{lib}/libsqlite3odbc-#{version}.dylib", "#{lib}/libsqlite3odbc-#{version}.so"
```

Comment:
I have contacted upstream, will see what they say. How would you feel about a symlink instead of a rename, so ODBC users at least have consistency in their config - and also things won't break if upstream fixes it?